Platform Support.md

```diff
@@ -0,0 +1,17 @@
+# Platform Support
+
+Being built on cargo and rustc, maturin is limited by [rust's platform support](https://doc.rust-lang.org/nightly/rustc/platform-support.html).
+
+**Automated tests**: On github actions, windows, mac and linux are tested, all on 64-bit x86. FreeBSD is also tested though circle ci, but might get removed at some point. Since CI is very time intensive to maintain, I'd like to stick to github action and these three platforms.
```

Comment:
Nit:

```suggestion
**Automated tests**: On github actions, windows, macOS and linux are tested, all on 64-bit x86. FreeBSD is also tested though circle ci, but might get removed at some point. Since CI is very time intensive to maintain, I'd like to stick to github action and these three platforms.
```